### DISCUSSION TITLE
Feature Request: Update the PocketBase version from the GUI

### DISCUSSION THREAD

**Discussion by miguelgargallo** [pocketbase/pocketbase#3159] [Q&A] [answered]
The idea behind that, is to stay up to date by pressing update inside the Admin panel, from 0.15.7 to 0.15.8 for example. It displays you the "What's new on that version" and if a compatibility or a future broken staff, a warning to "auto fix" by the PocketBase instance, or "Perform changes manually" with a guide from github PocketBase docs and instructions on GUI

**Comment by ganigeorgiev** [comment, marked as answer]
I have it in my todo but for now this is left aside as there is no easy way to trigger update from the UI when PocketBase is used as Go framework.

Additionally PocketBase is still not stable yet and an update could require some manual migration steps, including client-side code changes and at least for now it is better users to always check manually the changelog when performing an update.

**Comment by miguelgargallo** [comment]
Thanks, I am sure will be an awesome integration to those startups and entrepreneurs that joined that boat, to making things that easy as a button, thanks for your time, I appreciate it! <3
